hatchet_sdk/loader.py

```diff
@@ -207,10 +172,6 @@ def get_config_value(key, env_var):
             logger=defaults.logInterceptor,
             grpc_max_recv_message_length=grpc_max_recv_message_length,
             grpc_max_send_message_length=grpc_max_send_message_length,
-            otel_exporter_oltp_endpoint=otel_exporter_oltp_endpoint,
```

Comment:
do we want to print something and exit if these were previously set?

Reply:
yeah good call, I'll do that

Reply:
actually, I think we need to be more careful about this. Since this feature was a no-op if the env vars weren't set, I could imagine someone using the env vars in production but not having them in dev, and this causing a crash in production.

do you think a warning would be okay here? it's not as noisy unfortunately, but it also won't cause outages for our customers

Reply:
Yes, good catch